### DISCUSSION TITLE
Return data from axios response by default

### DISCUSSION THREAD

**Discussion by egreb** [orval-labs/orval#111] [Ideas]
Hello.

I was wondering if its possible to get data returned from the axios response by default?
Maybe this could be a configuration option.

Cheers

**Comment by anymaniax** [comment]
You can already do it but you need to use a mutator. check example [here](https://github.com/anymaniax/orval/blob/master/samples/react-app-with-react-query/src/api/mutator/custom-instance.ts) or the doc [here](https://orval.dev/reference/configuration#mutator)

**Reply by egreb** [reply]
I missed that in the documentation, sorry. Thanks.

**Reply by anymaniax** [reply]
No worries and don't hesitate if you have other questions
